### DISCUSSION TITLE
Hello! Problem connecting a Reddit account and the Bot: Account Connected: NONE

### DISCUSSION THREAD

**Discussion by UnlimitedWealthEmperor** [MalloyDelacroix/DownloaderForReddit#355] [Q&A]
Hello! When I connect an account it says that NONE account has been connected, so I can not use the bot.
did Anybody find a solution for this error?

![image](https://github.com/MalloyDelacroix/DownloaderForReddit/assets/126300019/0e20feb8-8dd8-4c86-9cb3-fadc51af8c90)

**Comment by cscott0108** [comment]
I'm getting the Invalid name stuff too... But I found that None account still works... I've authenticated as myself but it works... I want to figure out this Invalid Name because it happens all of the time... if I wait a few minutes, it starts working again.

**Comment by MalloyDelacroix** [comment]
This is due to reddit's new API changes.  They incorrectly rate limit the authentication request so that it returns an error.  This is why you are seeing None as the logged-in username.  This appears to be a problem on their end.  All I can do is wait for them to resolve it.
